examples/azure-identity/node/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG BUILDER=node:14
```

Comment:
```suggestion
ARG BUILDER=mcr.microsoft.com/mirror/docker/library/node:16
```

Reply:
updated in this and [here](https://github.com/Azure/azure-workload-identity/pull/580/files#diff-aec6fad245e042e4b50d8431d13942881c29de6060f9475485e292e2bada551f)